### ISSUE TITLE
How to save hanson after some changes without lost all comments ?

### ISSUE THREAD

**Issue by cuixiping** [timjansen/hanson#8]
Does hanson can only read but cannot write?

**Comment by timjansen** [comment]
Yes, that's right. You can, of course, write the data as JSON. But you will
lose all formatting and comments.

Reading and then writing Hanson without loosing formatting is very hard, if
there are non-trivial changes in the object. But even if it is only a
simple change in a number or string, this will be very confusing if there
is a comment that refers to that value. I wouldn't know a good way to write
a JSON file without user interaction.

Tim
